### ISSUE TITLE
Remove `rmi imagename` from uninstall logic

### ISSUE THREAD

**Issue by pravinpushkar** [dapr/cli#1093]
The `rmi imagename` behaviour is inconsistent across docker and different version for podman - https://github.com/dapr/cli/blob/3bf4b5ef737ebcf0750c7ad8ba28cf7474a3f65c/pkg/standalone/uninstall.go#L33

This logic was never working anyway even for docker also. So, I propose to remove this as logic as it is not much useful to remove the downloaded docker images.  

```
# Ubuntu 20 with Podman version 3.4.2
pravin@pravin-linux:~$ podman rmi -f daprio/dapr
Error: daprio/dapr: image not known
pravin@pravin-linux:~$ echo $?
1

# Macos with Podman version 4.2.1
-$ podman rmi -f daprio/dapr
-$ echo $?
0

```

## Release Note

RELEASE NOTE: **FIXED** Removed `rmi imagename` from `dapr uninstall` code flow

**Comment by mukundansundar** [comment]
For removing the image, the fully qualified name and tag were necessary ... Even without introduction of podman, this was failing silently since `docker rmi -f` does not return an error on image not being found. 
I agree that we can remove this logic since it was never working. We can look into adding the proper logic to remove older images of dapr in the future milestones.

**Comment by mukundansundar** [comment]
This is considered a P0 since it always fails with an error for `podman` v3 versions and it fails silently in v4 versions of podman which was what was running in macOs automated tests.

**Comment by shubham1172** [comment]
Closed by #1095
